packages/app-extensions/src/formField/editableValueFactory.js

```diff
@@ -95,8 +95,8 @@ const getOptions = (type, formField, modelField, utils) => {
       if (utils.intl) {
         options.intl = utils.intl
       }
-      options.postPointDigits = modelField.postPointDigits
-      options.maxValue = modelField.maxValue
+      options.postPointDigits = modelField.validation.decimalDigits.postPointDigits
```

Comment:
i would use lodash get so make this a bit safer. Otherwise an undefined validation object would throw en exception.